docs/docs/Configuration/configuration-cli.mdx

```diff
@@ -72,88 +87,167 @@ The following sections describe the available CLI commands and any non-default o
+  <TabItem value="uv (recommended)" label="uv (recommended)" default>
+
 ```bash
-langflow api-key
-# or
 uv run langflow api-key
 ```
 
+  </TabItem>
+  <TabItem value="Direct" label="Direct">
+
+```bash
+langflow api-key
+```
+
+  </TabItem>
+</Tabs>
+
+#### Options
+
+| Option | Default | Type | Description |
+|--------|---------|--------|-------------|
+| `--log-level` | `error` | String | Set the logging level. |
+
 ### langflow copy-db
 
-Copy the Langflow database files from the cache directory to the current directory containing `__main__.py`.
-You can find this directory by running `which langflow`.
+Copy the Langflow database files from the cache directory to the current Langflow installation directory, which is the directory containing `__main__.py`.
+You can find the copy target directory by running `which langflow`.
+
+The files copied are:
+
+* `langflow.db` - The main Langflow database, stored in the user cache directory.
+* `langflow-pre.db` - The pre-release database, if it exists.
+
+If these files don't exist in the cache directory, then nothing is copied.
+
+<Tabs groupId="Invocation">
+  <TabItem value="uv (recommended)" label="uv (recommended)" default>
+
+```bash
+uv run langflow copy-db
+```
+
+  </TabItem>
+  <TabItem value="Direct" label="Direct">
 
 ```bash
 langflow copy-db
-# or
-python -m langflow copy-db
 ```
 
-The database files are `langflow.db` and `langflow-pre.db`.
-If these files don't exist in the cache directory, then nothing is copied.
+  </TabItem>
+</Tabs>
 
 ### langflow migration
 
-Run or test database migrations:
+Manages database schema migrations for Langflow using [Alembic](https://alembic.sqlalchemy.org/en/latest/), a database migration tool for SQLAlchemy.
+
+The `migration` command has two modes:
+
+* Test mode checks if migrations can be applied safely without actually running the migrations.
+This is the default mode, and should be run before running `--fix`.
+
+* Fix mode actually applies the migrations to update your database schema.
+This is a destructive operation that can potentially delete data, so run `langflow migration --test` before running `--fix`.
+
+<Tabs groupId="Invocation">
+  <TabItem value="uv (recommended)" label="uv (recommended)" default>
+
+```bash
+uv run langflow migration [OPTIONS]
+```
+
+  </TabItem>
+  <TabItem value="Global" label="Global">
 
 ```bash
 langflow migration [OPTIONS]
-# or
-python -m langflow migration [OPTIONS]
 ```
 
+  </TabItem>
+</Tabs>
+
 #### Options
 
 | Option | Default | Type | Description |
 |--------|---------|--------|-------------|
-| `--test` | `true` | Boolean | Run migrations in test mode. Use `--no-test` to disable test mode. |
-| `--fix` | `false` (`--no-fix`) | Boolean | Fix migrations. This is a destructive operation, and all affected data will be deleted. Only use this option if you know what you are doing. |
+| `--test` | `test` | Boolean | Run migrations in test mode. Use `--no-test` to disable test mode. |
```

Comment:
does `no-test = --fix`, and does `no-fix = --test` ?

Reply:
Good question, I made a mistake here that `--test` is the default, so you actually run `langflow migration` with no flags, not `--test`.